### DISCUSSION TITLE
Geomap information

### DISCUSSION THREAD

**Discussion by korvinys095** [grafana/grafana#37803] [Dashboards, Panels & Visualizations]
Geomap it's beautiful, but it will be better if:
- It's show all information from Table. Some times i need to hide information. For example:
I have status like a word - Open, Close, In Work. I want to change color in different collor. I take status like number from database and use it in Unit. And i don't need this information.

But in this case it will be better, if Thresholds can take text (or i can choose another tool for change colors), and i put information
green - Close
red - Open
purple - In Work
Black - another
And in this case i will see more information on Legend!

- I can put different decimals, but if it like 9 - i can see 9.000000000. In casem i have status like numer (1,2,3) and LAT/LOG. It's better if i can see information like data from query, or if i can change decimals in table.

- In GeoMap i don't find, how make Sticky Labels. It will be better, if i have LAT / LONG in label, and i can click on point and copy information from Label.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
